saleor/graphql/webhook/schema.py

```diff
@@ -15,6 +16,17 @@ class WebhookQueries(graphene.ObjectType):
         description="Lookup a webhook by ID.",
     )
     webhooks = PrefetchingConnectionField(Webhook, description="List of webhooks")
+    webhook_sample_payload = graphene.Field(
+        graphene.JSONString,
+        event_type=graphene.Argument(
+            WebhookEventTypeEnum,
+            required=True,
+            description="Name of the reguested event type.",
```

Comment:
Please don't try to guess the event names.
```suggestion
            description="Name of the requested event type.",
```